src/main.tsx

```diff
@@ -13,9 +14,7 @@ ReactDOM.createRoot(document.getElementById("root")!).render(
   <React.StrictMode>
     <ApolloProvider client={client}>
       <BrowserRouter>
-        <Routes>
```

Comment:
I believe you should add this back, and then import Form here, as an element of this route as per my other comment.